addons/source-python/data/source-python/entities/orangebox/dod/CDODPlayer.ini

```diff
@@ -1,6 +1,7 @@
 [property]
 
     ragdoll = m_hRagdoll
+    desired_player_class = m_Shared.dodsharedlocaldata.m_iDesiredPlayerClass
```

Comment:
This shouldn't be required as it's implemented on C++ side. The rest looks good, thank you!

Reply:
Done.